control/control.go

```diff
@@ -94,8 +88,8 @@ func (srv *HTTPServer) Run(bus *events.EventBus) {
 
 // Start sets up API routes with the event bus, listens on the control
 // socket, and serves the HTTP server.
-func (srv *HTTPServer) Start() {
-	endpoints := &Endpoints{srv.Bus}
+func (srv *HTTPServer) Start(bus *events.EventBus) {
+	endpoints := &Endpoints{bus}
```

Comment:
Ah, that's a good point. I'd forgotten that we publish events through the endpoints.